sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/highlighting/XMLHighlighting.java

```diff
@@ -64,7 +81,7 @@ public void highlightXML(Reader reader) throws XMLStreamException {
     while (xmlReader.hasNext()) {
       Location prevLocation = xmlReader.getLocation();
       xmlReader.next();
-      int startOffset = xmlReader.getLocation().getCharacterOffset();
+      int startOffset = xmlReader.getLocation().getCharacterOffset() ;//+ offsetDelta;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ATrailingCommentCheck)